packages/hooks/src/useGrowthbookFeatureFlag.ts

```diff
@@ -1,30 +1,56 @@
 import { useState, useEffect } from 'react';
 import { Analytics } from '@deriv-com/analytics';
+import { useRemoteConfig } from '@deriv/api';
 
 interface UseGrowthbookFeatureFlagArgs<T> {
     featureFlag: string;
-    defaultValue: T;
+    defaultValue?: T;
 }
 
 const useGrowthbookFeatureFlag = <T extends string | boolean>({
     featureFlag,
     defaultValue,
 }: UseGrowthbookFeatureFlagArgs<T>) => {
-    const [featureFlagValue, setFeatureFlagValue] = useState<T>(defaultValue);
+    const resolvedDefaultValue: T = defaultValue !== undefined ? defaultValue : (false as T);
+    const [featureFlagValue, setFeatureFlagValue] = useState(
+        Analytics?.getFeatureValue(featureFlag, resolvedDefaultValue) ?? resolvedDefaultValue
+    );
+    const [isGBLoaded, setIsGBLoaded] = useState(false);
```

Comment:
@ali-hosseini-deriv just checking why we didnt use useIsGrowthbookIsLoaded here

Reply:
you are right @amina-deriv 
Just used it.